stock_reserve/model/stock_reserve.py

```diff
@@ -0,0 +1,197 @@
+##############################################################################
+#
+#    Author: Guewen Baconnier
+#    Copyright 2013 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from odoo import models, fields, api, _
+from odoo.exceptions import except_orm
```

Comment:
except_orm is deprecated, use UserError or Warning or any that suits your needs from the new api

Reply:
Is ValidationError the suitable new api?

Reply:
Yes, but is this a validation error?

Reply:
I an not sure. What's your advice?

Reply:
you can remove the import line and leave only the ValueError on line 118
`except ValueError:`

Because your are not raising the error but catching it.